src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -155,6 +157,34 @@ class IOUAmountPage extends React.Component {
         });
     }
 
+    /**
+     * Handle long press key on number pad.
+     * Only handles the '<' key and starts the continuous input timer.
+     *
+     * @param {String} key
+     */
+    handleLongPressNumberPad(key) {
```

Comment:
Hmmm shouldn't all this logic be on the BigNumberPad component? Do we expect any BigNumberPad were we don't want long presses to work? I would think we always want this behavior, so reimplementing it on each caller sounds worse than just incorporating it to that component.

Reply:
Good point, I was thinking the same but had doubts.

Reply:
Basically, my point was Continues keypress event timer logic should be inside the BigNumberPad. This callback should just be doing side-effect.

Reply:
But that component doesn't have any state itself, it is simply responsible to passing the callbacks to the buttons.

Or do you think we should pass some 'deleteSingleChar' prop to the number pad and call it with `setInterval` inside the `BigNumberPad`?

Reply:
Yup, so you just need to trigger the passed callback as you do now. But longPress will trigger that multiple times.

Reply:
Ah, I got you. There is no need for a new prop, I need to call the one that is already used for single press. :+1:

Reply:
@parasharrajat @iwiznia 
I have updated the PR with the change you requested, please review.